packages/jupyterlab-chat-extension/src/index.ts

```diff
@@ -618,8 +622,9 @@ const chatCommands: JupyterFrontEndPlugin<void> = {
               factory.tracker.add(widget);
             } else {
               // The chat is opened in the main area
+              // TODO: support JCollab v3 by optionally prefixing 'RTC:'
```

Comment:
Is this possible ?
The plugin requires `ICollaborativeContentProvider`, which is only available in jupyter_collaboration >= 4 as far as I know.

Reply:
jupyter-collaboration v4 introduces `ICollaborativeContentProvider` to be used instead of `ICollaborativeDrive` (see jupyter-collaboration [PR #418](https://github.com/jupyterlab/jupyter-collaboration/commit/14c14880b57d6f32b3c86fe8646c5a7480a781c4#diff-86381e434c47b18a3d8f457d3c20af61f75031ee9ecca0e12bcb165beea0dbbeL13)). 

This would require some effort but we could potentially check which jupyter_collaboration version is installed and use appropriate classes. This can be wrapped into a class or a shim to make it more straightowrward from the code readability POV.